src/game/models/asteroid.rs

```diff
@@ -144,6 +170,81 @@ impl Asteroid {
             on_screen: false,
         }
     }
+
+    pub fn can_split(&self) -> bool {
+        self.shape.len() > 10
+    }
+
+    pub fn split<P: Positioned>(&mut self, other: &P) -> Vec<Asteroid> {
+        self.normalize_rotation();
+        //let base_speed = self.vel();
+        let index_nearest = self.index_nearest_point(other);
+        let num_pieces = 3;
```

Comment:
Doing it later is totally fine, and you're right probably a good idea so we have a commit we can refer back to that has the minimal work on it.